src/Engine/Audio/SteamAudioDirectRenderer.cpp

```diff
@@ -1,11 +1,13 @@
 #include "SteamAudioDirectRenderer.hpp"
 
 SteamAudioDirectRenderer::SteamAudioDirectRenderer() {
-
+    context = nullptr;
+    environment = nullptr;
 }
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'SteamAudioDirectRenderer::environment' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)